### DISCUSSION TITLE
Drill-down via click on cell value

### DISCUSSION THREAD

**Discussion by darioguarascio** [directus/directus#12512] [Feature Requests]
I am a big user of Kibana, one of the features I use the most is the `Filter for Value`, a `(+)` icon appearing when hovering a table cell, which basically generates a `field: cell-value` filter and automatically applies it.

I was wondering if you ever considered such feature, navigating big tables would become much easier. 

The same result is of course possible to manually add a filter, selecting the field, the operator, and inputing the value, but we are all fans of faster shortcuts!

**Comment by benhaynes** [comment]
I like it! We're all about intelligent shortcuts too. So, you're thinking the user would right-click a "cell" in the table layout, and the context menu would show "Filter on this value"... which would add `[field] _eq [value]` to the filter?

**Comment by rijkvanzanten** [comment]
I'm a little hesitant to add it as a button on hover (which also means you have the "click on a row that's clickable" UX issue), but that would be a great candidate for a context menu (right click)

**Reply by darioguarascio** [reply]
@rijkvanzanten A context menu would also work ( 2 clicks instead of 1, but that's reasonable ). I don't know about UX issue, kibana uses it as an small icon right-aligned to the cell margin

@benhaynes yes, and because of the context menu it could be able to go full featured and have 
- Filter for value
- Filter out
- (if numerical / date / etc) >
  -- Greater than
  -- Less than
  
  and so on

**Reply by rijkvanzanten** [reply]
> I don't know about UX issue, kibana uses it as an small icon right-aligned to the cell margin

The main thing is that the whole row is clickable, so any clickable element within that row can easily cause confusion (a couple px too far next to the icon, and you open the row rather than toggle the icon)

**Reply by darioguarascio** [reply]
@rijkvanzanten makes sense, I totally see the point. Well, since I've seen an adoption of the context menu in different places, this case fits too. The only thing I doubt about context menus in general, in web pages at least, is that they are unexpected to be there.
I wonder what could be a hint of their existence.
I remember a while ago i found myself stuck because I couldn't even think of the right click option.
The other point would be that context menus are not really triggerable from mobile, but I guess this can be overlooked, not sure if the typical use case of directus is a desktop user but my guts tell me it is

**Reply by benhaynes** [reply]
I think right-click will be more obvious once it's thoroughly supported throughout the App and we mention it in our upcoming "Onboarding" process.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
